src/user.rs

```diff
@@ -0,0 +1,46 @@
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::{Path, PathBuf};
+
+use rustc_serialize::json;
+
+use project_types::Project;
+use Error;
+
+
+pub fn new_user<P: AsRef<Path>>(project: String, public: P, name: String) -> Result<(), Error> {
```

Comment:
iii. Does this mean that Project and User should have values of type `&String` instead of `String`? That is leading to a lot of manual lifetime managing...